x-pack/legacy/plugins/apm/public/components/shared/charts/Timeline/Marker/ErrorMarker.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPopover } from '@elastic/eui';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import {
+  TRACE_ID,
+  TRANSACTION_ID
+} from '../../../../../../common/elasticsearch_fieldnames';
+import { useUrlParams } from '../../../../../hooks/useUrlParams';
+import { fontSizes, px, unit } from '../../../../../style/variables';
+import { asDuration } from '../../../../../utils/formatters';
+import { ErrorDetailLink } from '../../../Links/apm/ErrorDetailLink';
+import { Legend, Shape } from '../../Legend';
+import { IWaterfallItemError } from '../../../../app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers';
+
+interface Props {
+  mark: IWaterfallItemError;
+}
+
+const Popover = styled.div`
+  max-width: ${px(280)};
+`;
+
+const Label = styled.div`
+  font-size: ${fontSizes.small};
+  padding-bottom: ${px(unit)};
+  &:last-of-type {
+    padding-bottom: 0;
+  }
+`;
+
+const Link = styled.div`
```

Comment:
Instead of creating a styled component for this, would it be possible to simply use the [`EuiLink`](https://elastic.github.io/eui/#/navigation/link) component?

Reply:
@formgeist I refactored it a bit, I'm styling the `ErrorDetailLink` component only to apply the correct margins, and it wraps the EuiLink(https://github.com/cauemarcondes/kibana/blob/waterfall-erros-timeline/x-pack/legacy/plugins/apm/public/components/shared/Links/apm/APMLink.tsx#L54)